src/index.js

```diff
@@ -0,0 +1,24 @@
+const initApp = () => {
+  console.log('hellow world');
+}
+
+initApp();
+
+const shareData = {
+  title: 'MDN',
+  text: 'Learn web development on MDN!',
+  url: 'https://developer.mozilla.org',
+}
+
+const btn = document.querySelector('button');
```

Comment:
Use css classes instead of element tags
```suggestion
const shareApiButton = document.querySelector('button');
```

Reply:
Done